Letterbook.Adapter.TimescaleFeeds/EntityConfig/ConfigureTimelinePost.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Letterbook.Adapter.TimescaleFeeds.EntityModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Letterbook.Adapter.TimescaleFeeds.EntityConfig;
+
+public class ConfigureTimelinePost : IEntityTypeConfiguration<TimelinePost>
+{
+	public void Configure(EntityTypeBuilder<TimelinePost> builder)
+	{
+		// This key is purely for EFCore's benefit, we'll likely never (need to) query on it
+		// TODO(profiling): Evaluate the performance impact of this key, and compare with the hastle of using `ExecuteSqlAsync` to do
+		// custom INSERTs
+		// If ExecuteInsert() ever happens, use that instead and drop this index
+		// https://github.com/dotnet/efcore/issues/29897
+		// Also, explore using BulkExtensions https://www.nuget.org/packages/EFCore.BulkExtensions
+		builder.HasKey(model => new { model.Time, model.PostId, model.AudienceId });
```

Comment:
if we don't use this key directly, then should we just use an auto-increment INT key or something? That seems like it will be more efficient than a compound key.

Reply:
I expect this to be a very insert heavy table, and I worry about using db-generated keys, because I think that's more likely to produce conflicts/add load.

The best thing might be a Guid key that we just never use.